pymeasure/instruments/hp/hp33120A.py

```diff
@@ -112,6 +112,54 @@ def __init__(self, adapter, name="Hewlett Packard 33120A Function Generator", **
         values=AMPLITUDE_UNITS,
         map_values=True
     )
+    burst_state = Instrument.control(
+        "BM:STATE?", "BM:STATE %d",
+        """Enable or disable burst modulation""",
+        validator=strict_discrete_set,
+        values=[0, 1],
+    )
+    burst_source = Instrument.control(
```

Comment:
Please add a new line between two definitions.

Reply:
resolved in [900d4a1](https://github.com/pymeasure/pymeasure/pull/1056/commits/900d4a112c5216f07a61a9f97cf3e66c791424df)